### PR TITLE
Missing HostReadWrite: pgridfunc.cpp::SaveAsOne 

### PR DESCRIPTION
**Pull request by artv3**
Bugfix when using SaveAsOne in cuda mode. 

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#1576](https://github.com/mfem/mfem/pull/1576) | @artv3 | @v-dobrev | @v-dobrev + @tzanio | 06/27/20 | 07/01/20 | 07/09/20 | |

### CODE REVIEW

File: fem/pgridfunc.cpp [hunk 1]
```diff
@@ -539,6 +539,7 @@ void ParGridFunction::SaveAsOne(std::ostream &out)
    int *nfdofs = new int[NRanks];
    int *nrdofs = new int[NRanks];
 
+   data.ReadWrite(MemoryClass::HOST,data.Capacity());
```
Comment (v-dobrev) [inline fem/pgridfunc.cpp:542]: Why not just call `HostRead()`? Also do we need `ReadWrite`? Isn't `Read` sufficient?

Comment (artv3) [reply fem/pgridfunc.cpp]: Good point.

**Comment by v-dobrev** [comment]
LGTM

**Comment by v-dobrev** [comment]
Merged in `next` for testing...

**Pull request merged**